setup.py

```diff
@@ -50,17 +50,21 @@ def read_file(filename):
     url='https://github.com/beproud/bpmailer/',
     classifiers=[
         'Development Status :: 3 - Alpha',
-        'Environment :: Plugins',
-        'Framework :: Django',
-        'Intended Audience :: Developers',
         'License :: OSI Approved :: BSD License',
         'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.9',
+        'Framework:: Django',
+        'Framework:: Django :: 2.2',
+        'Intended Audience :: Developers',
+        'Environment :: Plugins',
         'Topic :: Software Development :: Libraries :: Python Modules',
     ],
     include_package_data=True,
     packages=find_packages(),
     namespace_packages=['beproud', 'beproud.django'],
-    install_requires=['Django>=1.11', 'six'],
+    install_requires=['Django>=2.2', 'six'],
```

Comment:
依存で必要とREADMEにもrequirements.txtにも記載しているので必要かと思います。
```suggestion
    install_requires=['Django>=2.2', 'six', 'celery'],
```